man/org.freedesktop.systemd1.xml

```diff
@@ -5454,6 +5458,10 @@ node /org/freedesktop/systemd1/unit/avahi_2ddaemon_2esocket {
       meaning as they have for the corresponding field of service units (see above). In addition to that,
       the value <literal>service-failed-permanent</literal> indicates that the service of this socket failed
       continuously.</para>
+
+      <para><varname>FlushPending</varname> indicates if the socket is flushed
```

Comment:
`<varname>FlushPending=</> specifies whether to flush the socket just before entering the listening state.
This setting only applies to sockets with <varname>Accept=</varname> ...`